scss/tech.scss

```diff
@@ -35,6 +35,12 @@
       z-index: -1;
     }
 
+    &:hover::after {
```

Comment:
You dont need to make any change here, but notice that you included the changes of the PR #7 , when it wasnt needed